client/elements/text/sc-segmented-text.html

```diff
@@ -101,7 +103,7 @@ <h3>{{ localize('couldNotFind') }}</h3>
        * @extends {Polymer.Element}
        * @appliesMixin Polymer.AppLocalizeBehavior
        */
-      class SCSegmentedText extends Polymer.mixinBehaviors([Polymer.AppLocalizeBehavior], ReduxMixin(Polymer.Element)) {
+      class SCSegmentedText extends SCTextFunctions {
```

Comment:
1. We can make `SCSegmentedText` and `SCSimpleText` subclasses of another class, but I can see two issues here:
* `the SCTextFunctions` class mirrors the functionality of `SCStaticPage`, so maybe `SCTextFunctions` should extend `SCStaticPage` and `SCStaticPage` should then be renamed to something like `SCLocalizedElement`?
* Class naming is a bit off - a class name should be singular and the subclass should be a "subtype" of the parent - example: "Dog" or "Cat" extends "Animal". In this case I think `SCSegmentedText` could extend `SCText`.

Reply:
I'm a bit unsure of what you're saying here. I just created a class to take care of the doubling up of various functions that are used exclusively in texts.